packages/inventory/src/components/InventoryList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,3 +1,52 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Inventory list Page renders when data is present: link to add new inventory item 1`] = `
+<a
+  href="/inventory/profile/b8a7998c-5df6-49eb-98e6-f0675db71848/item/add"
+  onClick={[Function]}
+>
+  <Button
+    block={false}
+    ghost={false}
+    htmlType="button"
+    loading={false}
+    size="large"
+    type="primary"
+  >
+    <Wave>
+      <button
+        className="ant-btn ant-btn-primary ant-btn-lg"
+        onClick={[Function]}
+        type="button"
+      >
+        <LoadingIcon
```

Comment:
Do you think it is worth it to have a mocked loading component so as to reduce the size of this snapshot, and others like it?

Reply:
@moshthepitt this is part of the `<Button>` component as exported from antd (the Loader stuff is encapsulated withing Button). I think its going to be more trouble than its worth to just mock out the loading `LoadingIcon`. cc @HaseemUlHaq